.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+  - "3.6"
```

Comment:
change to 3.5, we've modified the main dockerfile to use 3.5 becuase 3.6 not available on sandbox/Ubuntu 16.04